### PR TITLE
updated RU locale

### PR DESCRIPTION
**Pull request by tonymorony**

### CODE REVIEW

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts [hunk 1]
```diff
@@ -5,16 +5,18 @@
     <name>QPlatformTheme</name>
     <message>
         <source>&amp;Yes</source>
-        <translation>Yes</translation>
+        <translation>&amp;Да</translation>
```
Comment (tonymorony) [inline atomic_defi_design/assets/languages/atomic_defi_ru.ts:8]: should I write content here with or without `&` @ssakone  @Milerius ?

**Comment by smk762** [comment]
Still some English here -

image
image

Other places for login / logout or import/create wallet look ok.

**Comment by tonymorony** [comment]
> Still some English here -
> 
> image
> image
> 
> Other places for login / logout or import/create wallet look ok.

 I believe localisation for this label was not added properly:

Screenshot 2021-07-22 at 12 59 53

File: atomic_defi_design/assets/languages/atomic_defi_ru.ts (hunk 1 above)
Comment (Milerius) [reply atomic_defi_design/assets/languages/atomic_defi_ru.ts:8]: without &

Comment (tonymorony) [reply atomic_defi_design/assets/languages/atomic_defi_ru.ts:8]: image

should I ignore the warning?

**Comment by tonymorony** [comment]
image
image

problems raised above seems to be fixed

**Pull request merged**